src/resources/Catalogs/CatalogInterfaces.ts

```diff
@@ -324,15 +324,42 @@ export type ScopeModel =
           sourceIds: string[];
       };
 
+export interface CatalogFieldStatsModel {
+    /**
+     * List of field statistics
+     */
+    fieldStats: FieldStatsModel[];
+}
+
+export type CachedCatalogFieldStatsModel = Cached<CatalogFieldStatsModel>;
```

Comment:
This does not align with the actual response of the endpoint.
```
{
  "fieldStats": [
    {
      "availability": {
        "objectsWithField": 9,
        "objectsWithType": 100
      },
      "fieldName": "ec_brand",
      "product": {
        "objectsWithField": 9,
        "objectsWithType": 100
      },
      "variant": {
        "objectsWithField": 9,
        "objectsWithType": 100
      }
    }
  ],
  "lastUpdated": "2022-03-31T13:05:10.858Z",
  "nextUpdate": "2022-03-31T13:05:10.858Z"
}
```

The `Cached<CatalogFieldStatsModel>` makes it expect:
```
{
  "item": {
    "fieldStats": [
      {
        "availability": {
          "objectsWithField": 9,
          "objectsWithType": 100
        },
        "fieldName": "ec_brand",
        "product": {
          "objectsWithField": 9,
          "objectsWithType": 100
        },
        "variant": {
          "objectsWithField": 9,
          "objectsWithType": 100
        }
      }
    ]
  },
  "lastUpdated": "2022-03-31T13:05:10.858Z",
  "nextUpdate": "2022-03-31T13:05:10.858Z"
}
```

Reply:
I see, it has been updated. I'll create another commit to align it.